packages/expo-audio/android/src/main/java/expo/modules/audio/AudioModule.kt

```diff
@@ -465,10 +465,12 @@ class AudioModule : Module() {
         recorder.prepareRecording(options)
       }
 
-      Function("record") { recorder: AudioRecorder ->
+      Function("record") { recorder: AudioRecorder, options: Map<String, Any>? ->
```

Comment:
```suggestion
      Function("record") { recorder: AudioRecorder, options: RecordOptions? ->
```

```swift
struct RecordOptions: Record {
  @Field var forDuration: Double?
  @Field var atTime: Double?
}
```
Let's create a record for this